src/targets/gpu/lowering.cpp

```diff
@@ -184,9 +185,16 @@ struct miopen_apply
         add_extend_op<hip_reduce_min, op::reduce_min>("reduce_min");
         add_extend_op<hip_reduce_prod, op::reduce_prod>("reduce_prod");
         add_extend_op<hip_reduce_sum, op::reduce_sum>("reduce_sum");
+        add_extend_op<hip_rnn_var_sl_shift_output, op::rnn_var_sl_shift_output>(
+            "rnn_var_sl_shift_output");
+        add_extend_op<hip_rnn_var_sl_shift_sequence, op::rnn_var_sl_shift_sequence>(
+            "rnn_var_sl_shift_sequence");
+        add_extend_op<hip_rnn_var_sl_last_output<op::rnn_last_cell_output>,
+                      op::rnn_last_cell_output>("rnn_last_cell_output");
+        add_extend_op<hip_rnn_var_sl_last_output<op::rnn_last_hs_output>, op::rnn_last_hs_output>(
+            "rnn_last_hs_output");
```

Comment:
`rnn_last_hs_output` and `rnn_last_cell_output` should be replaced in rewrite_rnn.